Source/WTF/wtf/fast_float/ascii_number.h

```diff
@@ -0,0 +1,231 @@
+#ifndef FASTFLOAT_ASCII_NUMBER_H
+#define FASTFLOAT_ASCII_NUMBER_H
+
+#include <cctype>
+#include <cstdint>
+#include <cstring>
+#include <iterator>
+
+#include <wtf/fast_float/float_common.h>
+
+namespace fast_float {
```

Comment:
For whatever reason, we wrapped all the double_conversion namespaces in an our `namespace WTF`. Not sure it makes a difference given this is all only included in one cpp file, but maybe for mangling reasons it would be a good idea?